### DISCUSSION TITLE
Can you stream a song from the server when that track is downloaded locally?

### DISCUSSION THREAD

**Discussion by Zedniac** [UnicornsOnLSD/finamp#970] [General]
While out and about, the default behavior of Finamp works great - it plays my downloaded files first, and only pulls them from the Jellyfin server when the track is not downloaded. However, when I'm at home (or somewhere with a fast connection) I often want to listen to the copy on the server since it would be direct played lossless rather than the local transcoded copy. 

As far as I can tell, there isn't a way to direct play the lossless file unless I delete the downloaded file. Am I missing this option somewhere?

If not, I would love an option where you can tell Finamp whether to default to the downloaded file (one option could be the current behavior, the other could be to always play from the server unless the connection is not fast enough to load the file in time to play it).

iPhone, Finamp 0.9.12 (Testflight Beta)

**Comment by Chaphasilor** [comment]
Hey @Zedniac, yes that's currently not possible, but there are plans to add support for different server URLs depending on WiFi connection status, and we could probably also add two sets of transcoding settings in that case!

**Reply by Zedniac** [reply]
Sweet, that would be very useful for me so if it’s considered I appreciate it. I don’t know of other apps that have this option, so either way I’ll be sticking with Finamp. Easily the best music streaming solution I’ve found. Thanks!
